lib/matplotlib/axes/_base.py

```diff
@@ -3283,8 +3284,11 @@ def ticklabel_format(self, *, axis='both', style='', scilimits=None,
             except (ValueError, TypeError) as err:
                 raise ValueError("scilimits must be a sequence of 2 integers"
                                  ) from err
-        STYLES = {'sci': True, 'scientific': True, 'plain': False, '': None}
-        is_sci_style = _api.check_getitem(STYLES, style=style)
+        STYLES = {'sci': True, 'scientific': True, 'plain': False}
+        if style is None:
+            is_sci_style = False
```

Comment:
```suggestion
            is_sci_style = None
```
`None` here should mean that the style is not changed from what it previously was.  Setting `is_sci_style` also to `None` achieves that.

Reply:
I'm changing it so that everything is in the dictionary. I believe it solves it more effectively. About setting it to None or False, I thought this variable should be either True or False but, after reading the logic after it, I've seen it can be None, indeed.